.dockerignore

```diff
@@ -1,4 +1,4 @@
-.git/
+# .git/
```

Comment:
?

Reply:
ah yes, I was going to ask about this. Git history is required to set the timestamps of the files. Is there any reason this directory **shouldn't** be included?

Reply:
the initial reason to include the .git in dockerignore is to only compare the files that matter for maximizing cache reusability.
If it actually helps with it then we should comment it out